Integrations/AlienVault_OTX_v2/AlienVault_OTX_v2.yml

```
@@ -0,0 +1,468 @@
+      description: AlienVault IP reputation
+      type: String
+    - contextPath: DBotScore.Indicator
+      description: The indicator we tested
+      type: String
+    - contextPath: DBotScore.Score
+      description: The score from AlienVault
+      type: Number
+    - contextPath: DBotScore.Type
+      description: Indicator type
+      type: String
+    - contextPath: DBotScore.Vendor
+      description: AlienVault vendor
+      type: String
+  - arguments:
+    - default: false
+      description: hostname to query
+      isArray: false
+      name: hostname
+      required: true
+      secret: false
+    deprecated: false
+    description: Query hostname in AlienVault OTX
+    execution: false
+    name: alienvault-search-hostname
+    outputs:
+    - contextPath: Endpoint.Hostname
+      description: Endpoint hostname
+      type: String
+    - contextPath: Endpoint.AlienVaultOTX.Alexa
+      description: Endpoint Alexa URL
+      type: String
+    - contextPath: Endpoint.AlienVaultOTX.Whois
+      description: Endpoint Whois URL
+      type: String
+    - contextPath: DBotScore.Score
+      description: The score from AlienVault
+      type: Number
+    - contextPath: DBotScore.Type
+      description: Indicator type
+      type: String
+    - contextPath: DBotScore.Vendor
+      description: AlienVault vendor
+      type: String
+    - contextPath: DBotScore.Indicator
+      description: The indicator we tested
+      type: String
+  - arguments:
+    - default: false
+      description: File hash to query
+      isArray: false
+      name: file
+      required: true
+      secret: false
+    deprecated: false
+    description: Query file in AlienVault OTX
+    execution: false
+    name: file
+    outputs:
+    - contextPath: File.MD5
+      description: MD5 Hash of the file
+      type: String
+    - contextPath: File.SHA1
+      description: SHA1 Hash of the file
+      type: String
+    - contextPath: File.SHA256
+      description: SHA256 Hash of the file
+      type: String
+    - contextPath: DBotScore.Indicator
+      description: The indicator we tested
+      type: String
+    - contextPath: DBotScore.Score
+      description: The score from AlienVault
+      type: Number
+    - contextPath: DBotScore.Type
+      description: Indicator type
+      type: String
+    - contextPath: DBotScore.Vendor
+      description: AlienVault vendor
+      type: String
+    - contextPath: File.Malicious.PulseIDs
+      description: IDs of pulses marked this as malicious
+      type: String
+    - contextPath: File.Type
+      description: File.Type /*use file.class from the api*/
+      type: String
+    - contextPath: File.Size
+      description: Size of the file in bytes
+      type: String
+    - contextPath: File.SSDeep
+      description: The ssdeep value for the file
+      type: String
+  - arguments:
+    - default: false
+      description: The cve to query
+      isArray: false
+      name: cve-id
```

argument names should be in snake_case style
```suggestion
      name: cve_id
```